how-to/how-to-delete-local-cache.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How to Delete The Extension for Visual Studio and The Classic Windows Desktop Client Local Cache and Cookies
```

Comment:
My suggestion is to modify the order and shorten it a bit:
How to Delete the Local Cache and Cookies for the AppBuilder VS Extension and Windows Client

You can write the full names in the description/page_title